src/global_distrib/mod_global_distrib_receiver.erl

```diff
@@ -111,11 +111,12 @@ init({Ref, ranch_tcp, _Opts}) ->
 %% gen_server API
 %%--------------------------------------------------------------------
 
-handle_info({tcp, _Socket, RawData}, #state{socket = Socket, buffer = Buffer} = State) ->
+handle_info({Tag, _Socket, RawData}, #state{socket = Socket, buffer = Buffer} = State)
+  when Tag == tcp; Tag == ssl ->
     do_setopts_and_receive_data(Socket, Buffer, RawData, State);
-handle_info({tcp_closed, _Socket}, State) ->
+handle_info({Tag, _Socket}, State) when Tag == tcp_closed; Tag == ssl_closed ->
```

Comment:
here with explicit tuples is better, than with `element(1, X)`

Reply:
This is because this case is treating `error` from `close` differently, and the size of the tuples are different. Anyway, can fix the other one 👌🏽